lib/remote-connector.js

```diff
@@ -101,6 +92,16 @@ RemoteConnector.prototype.resolve = function(Model) {
   });
 };
 
+RemoteConnector.prototype.resolve = function(Model) {
```

Comment:
Please move this method back to lines 79-86, to avoid unnecessary changes in the git history?

Reply:
Hmmm. How it possible if I moved a part of this method to the method before it (to the `define` function)?

Reply:
Ah, I see what you mean now. I extracted a new method `setupRemotingTypeFor` that allows us to keep the patch smaller - see 40adba0.

Now I am going to squash all commits into a single one, so that this pull request can be merged.

Reply:
Great! Thank you